test/test_helper.rb

```diff
@@ -31,6 +31,8 @@ class ActiveSupport::TestCase
   include ContentStoreHelpers
 
   setup do
+    stub_request(:get, %r{#{Plek.new.find("static")}/templates/locales/(en|cy)})
```

Comment:
1. I've added some more comments about why we remove this key
2. I've removed this since it's not needed